src/main/java/org/rundeck/plugins/S3LogFileStoragePlugin.java

```diff
@@ -95,6 +96,12 @@
             description = "Whether to force use of Signature Version 4 authentication. Default: false",
             defaultValue = "false")
     private String forceSigV4;
+    
+    @PluginProperty(
+            title = "Use Signature v2",
+            description = "Use of Signature Version 2 authentication for old container. Default: false",
+            defaultValue = "false")
+    private String useSigV2;
```

Comment:
you can change this to a `boolean`

Reply:
I don't understand why you want a boolean on this parameter. In this case, perhaps it will be better to change the variable "forceSigV4" in boolean too

Reply:
Done